conversant-htb-utils.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
```

Comment:
Some refactoring will need to be done. Please remove this file and instead we recommend the following:

1. Move the navigator function in to the main file: conversant-htb-validator.js
2. Utilize `Browser.topWindow` instead for the widow

Reply:
Thanks!  Having Browser.topWindow would make things a lot easier.  It doesn't seem to be documented, though.  Could you verify it's just a call with no argument?

Reply:
Oh topWindow is a property.  Perhaps I can add it to the stub as well.